bindings/python/src/torrent_info.cpp

```diff
@@ -307,6 +307,14 @@ void bind_torrent_info()
 
     class_<announce_entry>("announce_entry", init<std::string const&>())
         .def_readwrite("url", &announce_entry::url)
```

Comment:
interesting that this is read_write. maybe in 1.1 this class can be used to add trackers as well, is that the case?